src/index.html

```diff
@@ -9,10 +9,44 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
since `sun` and `31` are modifiers
```suggestion
    <div class="calendar calendar--start-day--sun calendar--month-length--31">
```